QA/PoroDevTest/src/main/java/common/ui_setup/pom_setup/BasePage.java

```diff
@@ -1,16 +1,59 @@
 package common.ui_setup.pom_setup;
 
-import common.ui_setup.ElementControl;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.Select;
 
 public abstract class BasePage {
-    public ElementControl elementControl;
     protected WebDriver driver;
 
     public BasePage(WebDriver driver) {
         this.driver = driver;
-        elementControl = new ElementControl(driver);
         PageFactory.initElements(driver, this);
     }
-}
+
+    public static void clickElement(WebElement element) {
+        element.click();
+    }
+    public static void clearElement(WebElement element) {
```

Comment:
This one is overkill. You won't use element.clear() function without element.sendKeys(text) function. I am suggesting to delete this method, and put element.clear() method in "sendText" method, above element.sendKeys(text)